edx_proctoring/tests/test_views.py

```diff
@@ -760,9 +760,10 @@ def test_ineligible_for_onboarding_exam(self):
         Test that the request returns a 404 error if the user cannot view the onboarding exam
         """
         course_sections = self.course_scheduled_sections
-        del course_sections[BlockUsageLocator.from_string(self.onboarding_exam.content_id)]
+        accessible_sequences = list(self.course_scheduled_sections.keys())
```

Comment:
My understanding is that, for this scenario, the course_sections should contain exams that the user does not have access to. accessible_sequences should not contain exams the learner does not have access to. I changed the test to reflect that.